cli/tests/pcluster/schemas/test_imagebuilder_schema.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import yaml
+
+from common.utils import load_yaml
+from pcluster.schemas.imagebuilder_schema import ImageBuilderSchema
+
+
+@pytest.mark.parametrize("config_file_name", ["imagebuilder_schema_simple.yaml", "imagebuilder_schema_dev.yaml"])
+def test_imagebuilder_schema(test_datadir, config_file_name):
+    # Load imagebuilder model from Yaml file
+    input_yaml = load_yaml(test_datadir / config_file_name)
+    print(input_yaml)
+    imagebuilder_config = ImageBuilderSchema().load(input_yaml)
+    print(imagebuilder_config)
+
+    # Re-create Yaml file from model and compare content
+    output_json = ImageBuilderSchema().dump(imagebuilder_config)
+
+    # TODO: don't compare the default value in imagebuilder config (e.g. delete_on_termination=true for volume config)
+    # assert_that(json.dumps(input_yaml, sort_keys=True)).is_equal_to(json.dumps(output_json, sort_keys=True))
```

Comment:
to be re-enabled?

Reply:
It Will be enabled when I implement the implied default values in config layer.